engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -73,7 +76,7 @@ def __init__(self, api_url: str, api_token: str):
         super().__init__(api_url, api_token)
 
     def check_token(self) -> Tuple[Optional[Response], dict]:
-        return self.api_get("api/org")
+        return self.api_head("api/org")
```

Comment:
no need to do a `GET` here, `HEAD` gives us the info we need (ie. 404, 401, 403) and is faster/transmits less data